### PR TITLE
feat : add Button typography variants

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# feat : add button and typography variants

### Icon Buttons
image

### Floating Action Button
image

### Muted Text
image

### Centered Text
image

### CODE REVIEW

File: pages/button/button.css [hunk 1]
```diff
@@ -31,3 +42,30 @@
   background-color: var(--danger-color);
   color: var(--white-color);
 }
+/* icon buttons  */
+.btn i {
+  padding: 5px;
+  /* color: var(--white-color); */
+}
+/* floating button example used in HTML */
```
Comment (ruchiragoswami) [inline pages/button/button.css:48]: At most places there is one empty line after each code block. However, it is  missed at some places like line 44, 48, 68 and some more. Just my thought.

File: pages/button/button.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <title>starUI🌟 | Button</title>
 </head>
 <body>
-  <div class="body-center">
+  <div class="body-center" id="body">
```
Comment (ruchiragoswami) [inline pages/button/button.html]: I personally don't like to use keywords for ids and classes. Example <body> is a tag and id is also "body" here. I add some other word with the keyword to make it distinctive.

**Review by ruchiragoswami — APPROVED** [review]
Looks good to me. Sharing my thoughts with you as comments.

**Review by Ayushi20-19 — APPROVED** [review]
Some changes already mentioned by Ruchi, apart code looks good.

File: pages/button/button.html (hunk 2 above)
Comment (Kevin-Solomon) [reply pages/button/button.html]: > I personally don't like to use keywords for ids and classes. Example is a tag and id is also "body" here. I add some other word with the keyword to make it distinctive.

I will resolve these issues Ruchi thank you for pointing them out.

**Review by AtulPant2704 — APPROVED** [review]

**Pull request merged**